Framework/CurveFitting/src/Algorithms/LeBailFunction.cpp

```diff
@@ -800,9 +800,7 @@ void LeBailFunction::groupPeaks(
         if (thispeak_rightbound < rightpeak_leftbound) {
           // this peak and its right peak are well separated.
           // finish this group by a copy
-          vector<pair<double, IPowderDiffPeakFunction_sptr>> peakgroupcopy =
-              peakgroup;
-          peakgroupvec.push_back(peakgroupcopy);
+          peakgroupvec.push_back(peakgroup);
```

Comment:
I think this could be more efficient with
```c++
peakgroupvec.emplace_back();
peakgroupvec.back().swap(peakgroup);
```
Saves a vector copy and `peakgroup.clear()` just below, no?